packages/graphback-cli/src/commands/dbRecreate.ts

```diff
@@ -0,0 +1,26 @@
+import chalk from 'chalk';
+import { DropCreateDatabaseAlways } from 'graphback'
+import { ConfigBuilder } from '../config/ConfigBuilder';
+import { createDB, postCommandMessage } from '../helpers'
+
+export const command = 'db:recreate'
```

Comment:
`db:recreate` is not valid posix argument as `:` has special bash meaning. I would use a recreate-db or something like that.